copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    # Kubernetes settings
+    kubernetes_api_url: str = "https://kubernetes.default.svc"
+    kubernetes_token_path: str = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    
+    # Argo CD settings
+    argo_cd_url: str = os.getenv("ARGO_CD_URL", "https://argo-cd-server.argo-cd.svc")
+    argo_cd_username: str = os.getenv("ARGO_CD_USERNAME", "admin")
+    argo_cd_password: str = os.getenv("ARGO_CD_PASSWORD", "")  # No default for security
+    
+    # TVP settings
+    tvp_repo_url: str = os.getenv("TVP_REPO_URL", "git@github.com:your-org/kubernetes-apps.git")
+    tvp_repo_path: str = os.getenv("TVP_REPO_PATH", "/tmp/kubernetes-apps")
```

Ruff (0.8.2)

Probable insecure usage of temporary file or directory: "/tmp/kubernetes-apps"

(S108)